public/views/positionForm.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Position List</title>
+    <link rel="stylesheet" href="../styles/reset.css">
+    <link rel="stylesheet" href="../styles/styles.css">
+    <script src="../js/index.js"></script>
+    <script src="../js/positionForm.js"></script>
+    <script src="../js/positionForm.js"></script>
+</head>
+<body>
+    <header>
+        <a id="mindSetLogo" href="/"><span>Mind</span>SET</a>
+        <nav>
+          <a href="#">admins</a>
+          <a href="#">applications</a>
+          <a href="#">clients</a>
+          <a href="#">interviews</a>
+          <a id="positionsNav" href="../views/positionList.html">positions</a>
+          <a href="#">profiles</a>
+          <a href="#">psychologists</a>
+          <a href="#">sessions</a>
+        </nav>
+      </header>
+      <div class="container">
+        <h2 class="header">Position form</h2>
+        <form id="form">
+            <span>Client:</span>
+            <input id="client" placeholder="Client" required>
+            <span>Job Description:</span>
+            <input id="jobDescription" placeholder="Job Description" type="text">
+            <span>Vacancy:</span>
+            <input id="vacancy" placeholder="Vacancy" type="text">
+            <span>Professional Profiles:</span>
+            <input id="professionalProfiles" placeholder="Professional Profiles" type="text" required>
+            <span>Is Open?</span>
+            <input id="isOpen" placeholder="You must enter true or false values" type="text" required>
```

Comment:
this input should be a `checkbox` since you are using a boolean value

Reply:
done!